prov/shm/src/smr_ep.c

```diff
@@ -460,6 +480,8 @@ int smr_endpoint(struct fid_domain *domain, struct fi_info *info,
 	smr_init_queue(&ep->trecv_queue, smr_match_tagged);
 	smr_init_queue(&ep->unexp_queue, smr_match_unexp);
 
+	ep->min_multi_recv_size = SMR_INJECT_SIZE;
```

Comment:
I saw your response in email, but github tossed it into the abyss somewhere.  I think this is as good a default as any.  This is something that the app really needs to decide based on what size messages it uses.